pkgs/build-support/agda/lib.nix

```diff
@@ -6,7 +6,7 @@
   * interfaceFile "Everything.agda" == "Everything.agdai"
   * interfaceFile "src/Everything.lagda.tex" == "src/Everything.agdai"
   */
```

Comment:
The comment here needs updating, as well as the tests in `nixos/tests/agda.nix` (but those seem entirely useless, so probably just remove them).

Reply:
Good catch. I have fixed the comment and remove the tests (the `rm` should be test enough).